boot/zephyr/Kconfig

```diff
@@ -28,6 +27,19 @@ partition=MCUBOOT_PAD
 partition-size=0x200
 source "${ZEPHYR_BASE}/../nrf/subsys/partition_manager/Kconfig.template.partition_size"
 
+config PM_PARTITION_SIZE_MCUBOOT
+	hex "Flash space reserved for the MCUboot image"
+	default 0xc000 if !BOOT_MIN_PARTITION_SIZE
+	default FPROTECT_BLOCK_SIZE if SOC_NRF9160
+	# nRF5340 uses SPU to lock flash, which uses lockable regions of 16kB.
+	# This image cannot be shrunk into a single region of this size without
+	# enabling external crypto.
+	default FPROTECT_BLOCK_SIZE if SOC_NRF5340_CPUAPP && BOOT_NRF_EXTERNAL_CRYPTO
```

Comment:
The comment combined with this code sounds like the prompt should be disabled in this case, to avoid reconfiguration.
Remember, a default value is not locked, it's just a default if nothing else is configured.

Reply:
Broke this prompt up into chunks that are dependent upon the relevant Kconfig options. Hopefully this is more in line with what you suggest here.